<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(26, 29, 60);
color: rgb(255, 255, 255);
font: 18pt &quot;黑体&quot;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,12,1,1">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/img/icon/logo.png);
background-color: qlineargradient(spread:pad, x1:0.0113636, y1:0.011, x2:1, y2:0, stop:0 rgba(26, 29, 60, 255), stop:0.568182 rgba(57, 94, 150, 255), stop:0.75 rgba(57, 94, 150, 255), stop:1 rgba(26, 29, 60, 255));

border-radius:5px;
 
 </string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 75 40pt &quot;黑体&quot;;
color: rgb(4, 170, 255);

 </string>
       </property>
       <property name="text">
        <string>可视化废票信息传递系统</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCameraProp">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/icon/camera.ico);
border-radius:7px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetup">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/icon/setup.ico);
border-radius:7px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,0,3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="title">
          <string>工序信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" rowstretch="2,1,2,1" columnstretch="2,7,1" columnminimumwidth="0,0,0">
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
            </property>
            <property name="text">
             <string>机长</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbMachineLeader">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(48, 48, 85);
border-radius:7px;
border:1px solid green;
color: rgb(0, 255, 255);
font: 36pt &quot;幼圆&quot;;</string>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true"> 

font: 75 14pt &quot;微软雅黑&quot;;
 </string>
            </property>
            <property name="text">
             <string>工序</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbWorkSeq">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(48, 48, 85);
font: 36pt &quot;幼圆&quot;;
color: rgb(0, 255, 255);
 
border-radius:7px;
border:1px solid green;</string>
            </property>
            <property name="text">
             <string>凹印</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>10</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>产品废票信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0" columnstretch="2,1,1,4">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <property name="horizontalSpacing">
             <number>1</number>
            </property>
            <property name="verticalSpacing">
             <number>4</number>
            </property>
            <item row="4" column="0">
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true"> border-top:1px solid rgb(122,122,122);</string>
              </property>
              <property name="text">
               <string>说明</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_11">
              <property name="styleSheet">
               <string notr="true">border-bottom:1px solid white;</string>
              </property>
              <property name="text">
               <string>标记</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="comYear">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>4</width>
                <height>4</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 255);
font: 50pt &quot;黑体&quot;;
 
 

</string>
              </property>
              <property name="currentText">
               <string>23</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <property name="maxCount">
               <number>5</number>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtBottom</enum>
              </property>
              <item>
               <property name="text">
                <string>22</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>23</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>2</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true"> 
</string>
              </property>
              <property name="text">
               <string>开位</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">border-bottom:1px solid white;</string>
              </property>
              <property name="text">
               <string>产品年份</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>4</number>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="editProductType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>2</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 255);
font: 50pt &quot;黑体&quot;;
border:0px solid rgb(100,100,100);
</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="3">
             <widget class="QTextEdit" name="txtDescribe">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>3</verstretch>
               </sizepolicy>
              </property>
              <property name="placeholderText">
               <string>请输入说明</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QLineEdit" name="editK">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>3</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="mouseTracking">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(240, 240, 240);
 
color: rgb(81, 81, 81);
font: 40pt &quot;黑体&quot;;
border-top:1px solid white;</string>
              </property>
              <property name="maxLength">
               <number>2</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="placeholderText">
               <string>?k</string>
              </property>
              <property name="cursorMoveStyle">
               <enum>Qt::VisualMoveStyle</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="editProductID">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>3</horstretch>
                <verstretch>2</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">
 
color: rgb(0, 255, 255);
font: 50pt &quot;黑体&quot;;
border:0px solid rgb(100,100,100);
</string>
              </property>
              <property name="maxLength">
               <number>4</number>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_10">
              <property name="styleSheet">
               <string notr="true">border-bottom:1px solid white;</string>
              </property>
              <property name="text">
               <string>品种</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLineEdit" name="editFlag">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>2</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 255);
font: 50pt &quot;黑体&quot;;
border:0px solid rgb(100,100,100);
</string>
              </property>
              <property name="maxLength">
               <number>1</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">border-bottom:1px solid white;</string>
              </property>
              <property name="text">
               <string>车号</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="margin">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="4">
             <widget class="Line" name="line">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>1</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLabel" name="label_8">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 85, 0);</string>
              </property>
              <property name="text">
               <string>  &lt;--请录入开位</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,2">
       <item>
        <widget class="QLabel" name="lbCamera">
         <property name="styleSheet">
          <string notr="true">border:4px solid green;
background-color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#19c80c;&quot;&gt;未发现摄像头。。。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3" columnstretch="5,3,1,1,1,3,3,0" columnminimumwidth="0,0,0,0,0,0,0,0">
         <item row="1" column="0">
          <widget class="QPushButton" name="btnDetcCart">
           <property name="styleSheet">
            <string notr="true">:hover
{
 
 color: rgb(53, 161, 0);
 border: 1px solid rgb(15 , 150 , 255);
}
:pressed
{
 color:rgb(75, 75, 75);
 background-color:rgb(204 , 228 , 247);
  
 padding-left:3px;
 padding-top:3px;
}
QPushButton{background-color: qlineargradient(spread:pad, x1:0.488636, y1:0.898, x2:0.500318, y2:0.302, stop:0 rgba(181, 181, 181, 255), stop:0.948864 rgba(243, 243, 243, 255));
color: rgb(75, 75, 75);}</string>
           </property>
           <property name="text">
            <string>识别车号</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="btnUploadRIO">
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(0, 255, 0);
border-radius:7px;
height:35px;
}

:pressed
{ 
 padding-left:3px;
 padding-top:3px;
}
:hover
{
 color:rgb(75, 75, 75);
}</string>
           </property>
           <property name="text">
            <string>上传废票截图</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="7">
          <widget class="QListWidget" name="lsImgROIWidget"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbNumberpPic">
           <property name="styleSheet">
            <string notr="true">border:1px solid blue;
background-color: rgb(148, 148, 148);</string>
           </property>
           <property name="text">
            <string>车号</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnCounterRot">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>72</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="btnUpLocal">
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(0, 255, 0);
border-radius:7px;

}
:pressed
{ 
 padding-left:3px;
 padding-top:3px;
}
:hover
{
 color:rgb(75, 75, 75);
}</string>
           </property>
           <property name="text">
            <string>上传本地图片</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="btnViewROITable">
           <property name="styleSheet">
            <string notr="true">QPushButton{background-color: rgb(0, 255, 0);
border-radius:7px;
height:35px;
}
:pressed
{ 
 padding-left:3px;
 padding-top:3px;
}
:hover
{
 color:rgb(75, 75, 75);}


</string>
           </property>
           <property name="text">
            <string>ROI图查看</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="btnRot">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>72</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
